airbyte-integrations/base-python/airbyte_protocol/airbyte_protocol/__init__.py

```diff
@@ -1,4 +1,26 @@
 from typing import Generator
+import yaml
+import pkgutil
+import warnings
+import python_jsonschema_objects as pjs
+
+
+def _load_classes(yaml_path: str):
+    data = yaml.load(pkgutil.get_data(__name__, yaml_path), Loader=yaml.FullLoader)
+    builder = pjs.ObjectBuilder(data)
+    return builder.build_classes(standardize_names=False)
```

Comment:
Is there a reason to prefer dynamic bindings over static generation? Might make sense to retain the context in a comment

Reply:
The only static builders for Python I found were https://github.com/pearmaster/json-schema-codegen and https://github.com/microsoft/jschema-to-python, which aren't as full-featured or widely used.

I can add a todo  https://github.com/airbytehq/airbyte/issues/530